docs/pages/Kratos/For_Users/General/Overview.md

```diff
@@ -48,4 +48,4 @@ _Kratos_ is **OPEN SOURCE**. The main code and program structure is available an
 
 ![](https://raw.githubusercontent.com/KratosMultiphysics/Documentation/master/Wiki_files/Home/Manos.jpg)
 
-_Kratos_ is **FREE** because is devoted mainly to developers, researchers and students and, therefore, is the most fruitful way to share knowledge and built a robust numerical methods laboratory adapted to their users' needs. Free because you have the freedom to modify and distribute the software. The one thing you're not able to do with free software is take away other people's freedom. Please, read the license for more detailed information.
+_Kratos_ is **FREE** because is it devoted mainly to developers, researchers and students and, therefore, is the most fruitful way to share knowledge and built a robust numerical methods laboratory adapted to their users' needs. Free because you have the freedom to modify and distribute the software. The one thing you're not able to do with free software is take away other people's freedom. Please, read the license for more detailed information.
```

Comment:
Shouldn't it be "it is devoted" ?

Reply:
Yes. Fixed it in [4212bb6](https://github.com/KratosMultiphysics/Kratos/pull/10801/commits/4212bb6701338c74ea00ae39c7cbe1876b502ab4). Thank you!